### PR TITLE
Adding third party notices

### PR DESCRIPTION
**Pull request by kartang**

### CODE REVIEW

File: ThirdPartyNotices.txt [hunk 1]
```diff
@@ -0,0 +1,684 @@
...
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+=========================================
+END OF OpenCover NOTICES, INFORMATION, AND LICENSE
+
+
+%% Owin and WindowsAzure.Storage NOTICES, INFORMATION, AND LICENSE BEGIN HERE
```
Comment (kartang) [inline ThirdPartyNotices.txt]: Owin dependency is slightly confusing. Should it refer to the Microsoft.Owin as opposed to the Owin nupkg available?

File: ThirdPartyNotices.txt [hunk 2]
```diff
@@ -0,0 +1,684 @@
...
+(D) If you distribute any portion of the software, you must retain all copyright, patent, trademark, and attribution notices that are present in the software.
+(E) If you distribute any portion of the software in source code form, you may do so only under this license by including a complete copy of this license with your distribution. If you distribute any portion of the software in compiled or object code form, you may only do so under a license that complies with this license.
+(F) The software is licensed "as-is." You bear the risk of using it. The contributors give no express warranties, guarantees or conditions. You may have additional consumer rights under your local laws which this license cannot change. To the extent permitted under your local laws, the contributors exclude the implied warranties of merchantability, fitness for a particular purpose and non-infringement.
+=========================================
+END OF WiX NOTICES, INFORMATION, AND LICENSE
+
+%% xunit, AspNet.ScriptManager.jQuery and AspNet.ScriptManager.bootstrap NOTICES, INFORMATION, AND LICENSE BEGIN HERE
```
Comment (kartang) [inline ThirdPartyNotices.txt]: For AspNet.* dependencies, I just used the outercurve license, as I found it as the owner in nuget.org.

File: ThirdPartyNotices.txt [hunk 3]
```diff
@@ -0,0 +1,684 @@
...
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+=========================================
+END OF knockoutjs NOTICES, INFORMATION, AND LICENSE
+
+%% Modernizr NOTICES, INFORMATION, AND LICENSE BEGIN HERE
+=========================================
+The MIT License (MIT)
+
+Copyright 2009-2016
```
Comment (kartang) [inline ThirdPartyNotices.txt]: I didin't find the copyright owner for this dependency.

**Comment by abaranch** [comment]
:shipit:

File: ThirdPartyNotices.txt (hunk 2 above)
Comment (dnduffy) [inline ThirdPartyNotices.txt]: So it actually says it is the Apache License, interesting.  Are these files actually copyrighted by Outercurve?

File: ThirdPartyNotices.txt [hunk 4]
```diff
@@ -0,0 +1,684 @@
...
+18	OpenCover (https://github.com/opencover/opencover)
+19	Owin (https://github.com/owin-contrib/owin-hosting/)
+20	Respond (https://github.com/scottjehl/Respond/)
+21	Rx-Core (https://github.com/Reactive-Extensions/Rx.NET)
+22	Rx-Interfaces (https://github.com/Reactive-Extensions/Rx.NET)
+23	Rx-Linq (https://github.com/Reactive-Extensions/Rx.NET)
+24	StyleCop.MSBuild (https://github.com/adamralph/stylecop-msbuild)
+25	System.Spatial (http://odata.github.io/)
+26	WebGrease (http://webgrease.codeplex.com/)
+27	WindowsAzure.Storage (https://azure.microsoft.com/en-us/documentation/services/storage/)
```
Comment (dnduffy) [inline ThirdPartyNotices.txt]: How is this 3rd party if it comes from Microsoft?

**Comment by dnduffy** [comment]
:+1:

File: ThirdPartyNotices.txt (hunk 2 above)
Comment (kartang) [inline ThirdPartyNotices.txt]: The only info I got is from nugget.org where I found that the owners of AspNet.* as Outercurve. https://www.nuget.org/packages/AspNet.ScriptManager.jQuery/

File: ThirdPartyNotices.txt (hunk 4 above)
Comment (kartang) [inline ThirdPartyNotices.txt]: It did not start with Microsoft.*? I thought we still need to add a notice for that.

Comment (dnduffy) [inline ThirdPartyNotices.txt]: The URL is a Microsoft URL, so where does it come from?  Who is the publisher?

**Pull request merged**